bin/create-probot-app.js

```diff
@@ -29,6 +29,7 @@ program
   .option('--overwrite', 'Overwrite existing files', false)
   .option('--template <template-url>', 'URL of custom template',
     TEMPLATE_REPO_URL)
+  .option('--typescript', 'Use the TypeScript template', 'https://github.com/probot/ts-template.git')
```

Comment:
Can we add a shorthand `-ts` here?

Reply:
`-ts` is two letters, so we could do `-t` or just shorten the longform to `--ts`

Reply:
Oh I didn't realize shorthand had to be one. I'm fine with either one to be honest. Or neither. Up to you 🤷‍♂️